nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1301,19 +1301,6 @@ define(
                         case 'queued':
                             return { mode: 'processing', stage: 'queued' };
                         case 'running':
-                            // see if any subjobs are done, if so, set the stage to 'partial-complete'
-                            if (jobState.child_jobs && jobState.child_jobs.length) {
```

Comment:
this will never be true due to the way that the app cell batch mode represents batch jobs (i.e. it doesn't)